HousingRepairsOnlineApi.Tests/HelpersTests/NotificationConfigurationResolverTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Helpers;
+using HousingRepairsOnlineApi.Helpers.SendNotifications;
+using Moq;
+using Xunit;
+
+namespace HousingRepairsOnlineApi.Tests.HelpersTests;
+
+public class NotificationConfigurationResolverTests
+{
+    private readonly Mock<TenantNotificationConfigurationProvider> tenantNotificationConfigurationProvider;
+    private readonly NotificationConfigurationResolver systemUnderTest;
+
+    public NotificationConfigurationResolverTests()
+    {
+        tenantNotificationConfigurationProvider = new Mock<TenantNotificationConfigurationProvider>("", "", "");
+
+        Dictionary<string, INotificationConfigurationProvider> notificationConfigurationProviders =
+            new Dictionary<string, INotificationConfigurationProvider>();
+        notificationConfigurationProviders.Add("TENANT", tenantNotificationConfigurationProvider.Object);
+
+        systemUnderTest = new NotificationConfigurationResolver(notificationConfigurationProviders);
+    }
+
+    [Fact]
+    public void GivenInvalidRepairType_WhenResolving_ThenExceptionIsReturned()
```

Comment:
This title indicates this is testing an invalid repair type, but the parameter is `COMMUNAL` which is a valid [repair type](https://github.com/Newarkandsherwood/housing-repairs-online-api/blob/main/HousingRepairsOnlineApi/Helpers/RepairType.cs).

Would it be more appropriate to name this test `Given[Unregistered|Unresolvable]RepairType...`?

I feel that `GivenInvalidRepairType_WhenResolving_ThenExceptionIsReturned` is also a valid test, which would need to pass an invalid argument to the `Resolve` method.